index.html

```diff
@@ -0,0 +1,205 @@
+      <section class="images"><img src="http://i.imgur.com/IWS6ooU.png" title="source: imgur.com" /></a>
+      </section>
+      <section>
+        <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.</p>
+      </section>
+      <section class="available_">
+        <p>NOT AVAILABLE</p>
+      </section>
+<!-- PRODUCT 1 SECTION 2: section-> header(specifications)-> 2 block elemets-> "size" -> "weight" -> footer-> "spec valid + date" -->
+    <section>
+      <header class="spec_price">Specifications</header>
+        <p class="size_weight">Size: 12" x 12"</p>
+        <p class="size_weight">Weight: 180 gr.</p>
+      <footer class="fine_print">Specifications valid until 04/21/2017</footer>
+    </section>
+<!--PRODUCT 1 SECTION 3: section-> header-> 3 block elements-> each with info on pricing for different quantities -->
+    <section>
+      <header class="spec_price">Pricing</header>
+      <div>
+        <p class="prices">1: $25.99</p>
+        <p class="prices">2: $45.99</p>
+        <p class="prices">3: $68.99</p>
+      </div>  
+    </section>
+
+
+  </article>
+  <article class="article_2">
+    <header class="title_"><h1>Bryter Layter</h1></header>
+      <section class="images"><img src="http://i.imgur.com/jIVSmh3.png" title="source: imgur.com" /></a>
+      </section>
+      <section>
+        <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.</p>
+      </section>
+      <section class="available_">
+        <p>AVAILABLE</p>
+      </section>
+<!-- PRODUCT 1 SECTION 2: section-> header(specifications)-> 2 block elemets-> "size" -> "weight" -> footer-> "spec valid + date" -->
+    <section>
+      <header class="spec_price">Specifications</header>
+        <p class="size_weight">Size: 12" x 12"</p>
+        <p class="size_weight">Weight: 180 gr.</p>
+      <footer class="fine_print">Specifications valid until 04/21/2017</footer>
+    </section>
+<!--PRODUCT 1 SECTION 3: section-> header-> 3 block elements-> each with info on pricing for different quantities -->
+    <section>
+      <header class="spec_price">Pricing</header>
+      <div>
+        <p class="prices">1: $25.99</p>
+        <p class="prices">2: $45.99</p>
+        <p class="prices">3: $68.99</p>
+      </div>  
+    </section>
+
+
+  </article>
+  <article class="article_3">
+    <header class="title_"><h1>Remain In Light</h1></header>
+      <section class="images"><img src="http://i.imgur.com/WI4BYhE.png" height: 200px width 200px title="source: imgur.com" /></a>
```

Comment:
Your img tag is messed up here. I think you meant to do:
`<img src="http://i.imgur.com/WI4BYhE.png" style="height: 200px width: 200px" title="source: imgur.com" />`, but don't do this. Style your images in CSS, not inline.